craco.config.cjs

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { VanillaExtractPlugin } = require('@vanilla-extract/webpack-plugin')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+
+module.exports = {
+  babel: {
+    plugins: ['@vanilla-extract/babel-plugin'],
+  },
+  webpack: {
+    plugins: [new VanillaExtractPlugin()],
+    configure: (webpackConfig, { env }) => {
+      if (env === 'production') {
+        const instanceOfMiniCssExtractPlugin = webpackConfig.plugins.find(
+          (plugin) => plugin instanceof MiniCssExtractPlugin
+        )
+        instanceOfMiniCssExtractPlugin.options.ignoreOrder = true
```

Comment:
I was running into a where the css would build in a different order which would cause the build to fail but had no affect on the UX. I investigated the issue and found this thread: https://github.com/facebook/create-react-app/issues/5372 which included this solution which resolved the issue.

Reply:
Is it necessary to check for production, or can this be simplified by always setting `ignoreOrder = true`?

Reply:
`.options` only exists on a prod env instance of `webpackConfig` otherwise you get an undefined error